home.css

```diff
@@ -0,0 +1,95 @@
+@font-face {
+    font-family: "intro-inline";
+    src: url("resources/intro/Intro_Inline.otf");
```

Comment:
From what I was able to find, this font is only free for personal use. We need to replace it (and delete files related to it), or make sure it is actually free for non-personal use.

Reply:
The client has this font in their Logo, so either they already acquired the proper licences or they are using a different Font. Etiher way this may then be a placeholder for the time being, we would need more details from the client on that one. Since we are not publishing yet, there isnt an issue arrising just now. We need to discuss this with the client then